Example/QuranEngineApp/Info.plist

```diff
@@ -2,6 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLName</key>
+			<string>com.quran.QuranEngineApp</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>quran-ios</string>
```

Comment:
Let's add an entry for `quran` scheme please.

Reply:
addressed